saleor/graphql/product/tests/test_category.py

```diff
@@ -605,6 +666,65 @@ def test_category_update_mutation(
     assert data["category"]["backgroundImage"]["alt"] == image_alt
 
 
+@patch("saleor.plugins.webhook.plugin._get_webhooks_for_event")
+@patch("saleor.plugins.webhook.plugin.trigger_webhooks_async")
+def test_category_update_trigger_webhook(
+    mocked_webhook_trigger,
+    mocked_get_webhooks_for_event,
+    any_webhook,
+    monkeypatch,
+    staff_api_client,
+    category,
+    permission_manage_products,
+    media_root,
+    settings,
+):
+    mocked_get_webhooks_for_event.return_value = [any_webhook]
+    settings.PLUGINS = ["saleor.plugins.webhook.plugin.WebhookPlugin"]
+    mock_create_thumbnails = Mock(return_value=None)
+    monkeypatch.setattr(
+        (
+            "saleor.product.thumbnails."
+            "create_category_background_image_thumbnails.delay"
+        ),
+        mock_create_thumbnails,
+    )
+
+    category_name = "Updated name"
+    description = "description"
+    category_slug = slugify(category_name)
+    category_description = dummy_editorjs(description, True)
+
+    image_file, image_name = create_image()
+    image_alt = "Alt text for an image."
+
+    variables = {
+        "name": category_name,
+        "description": category_description,
+        "backgroundImage": image_name,
+        "backgroundImageAlt": image_alt,
+        "id": graphene.Node.to_global_id("Category", category.pk),
+        "slug": category_slug,
+    }
+    body = get_multipart_request_body(
+        MUTATION_CATEGORY_UPDATE_MUTATION, variables, image_file, image_name
+    )
+    response = staff_api_client.post_multipart(
+        body, permissions=[permission_manage_products]
+    )
+    content = get_graphql_content(response)
+    data = content["data"]["categoryUpdate"]
+    assert data["errors"] == []
+
+    mocked_webhook_trigger.assert_called_once_with(
```

Comment:
question: what if we update category with no changed values? Will webhook trigger?